wheel/.gitignore

```diff
@@ -1 +0,0 @@
-target
```

Comment:
you just added this file in a previous commit. I think it would be better to not add it at all if you're just removing it again